tests/common/ignore-options.test.ts

```diff
@@ -7,8 +7,8 @@ import {
   IgnorePatternOption,
   shouldIgnore
 } from "../../src/common/ignore-options";
-import { typescript } from "../helpers/configs";
-import { createDummyRule } from "../helpers/util";
+import { typescript, filename } from "../helpers/configs";
+import { createDummyRule, addFilename } from "../helpers/util";
```

Comment:
Really nitpicky but these probably should be alphabetically sorted 😛

Reply:
I guess there are linting rules for this :-). They should be sorted now.